src/app/recommended-communities/recommended-communities.component.spec.ts

```diff
@@ -113,6 +155,15 @@ describe('RecommendedCommunitiesComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  it('should remove the community form the list of recommended communities when user has joined to it', fakeAsync(() => {
```

Comment:
I think there is a misprint in the word `form`.

Reply:
thank you. done